.github/workflows/release.yaml

```diff
@@ -24,6 +24,32 @@ jobs:
               }
             })
 
+  build-rom-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📂 Checkout code
+        uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: 👷 Build prql-compiler
+        run: cargo build --release
+      - name: Copy files into .rpm package
+        run: |
+          mkdir -p .rpmpkg/usr/bin
+          cp target/release/prqlc .rpmpkg/usr/bin/prqlc
+          chmod +x .rpmpkg/usr/bin/prqlc
+      - name: 📦 Build .rpm package
+        uses: jiro4989/build-rpm-action@v2
+        with:
+          summary: PRQL Compiler
+          package: prqlc
+          package_root: .rpmpkg
+          maintainer: The PRQL Project
+          version: ${{ github.ref }}
```

Comment:
```suggestion
          version: ${{ github.ref_name }}
```

Unless I'm misunderstanding something (I might be) — this gives the actual version number

Reply:
Yeah, but `build-rpm-action` will strip the `refs/tags/` from it anyway, so it doesn't matter. The example at https://github.com/marketplace/actions/build-rpm-action uses `github.ref`.

Reply:
OK. I think the reason why they use `github.ref` is because that is older than `refname`, but you're right that they use it

Reply:
Do you want me to use `github.ref` or `github.refname`? Feel free to edit as you wish.

Reply:
My guess is that `refname` is more direct, but I'm not 100% sure. If `ref` works then it's OK!